docs/src/tutorials/warp.md

```diff
@@ -0,0 +1,61 @@
+# Training a Homography matrix
+
+Homography matrices are projective transformations. They can be represented using `DiffImages.Homography()` in Julia.
+
+In this example, we will train a homography matrix using DiffImages.jl.
+## Importing the libraries
+```julia
+using DiffImages, ImageCore, ImageTransformations, FileIO
+```
+
+## Loading the images
+Let us load the images first. We will also convert them to `Float32` precision type since we do not need such high precision.
+```julia
+img = load("docs/src/assets/source.jpg") .|> RGB{Float32} 
+tgt = load("docs/src/assets/target.jpg") .|> RGB{Float32}
```

Comment:
How would you ensure the codes are runnable? How would you make sure the images match the actual function output? How would you expect your docs reader to load an image that doesn't exist locally?

Try to use Documenter's `@example`/`@repl`/`@setup` for this. Or maybe just adopt Literate.jl.

Reply:
I have changed it to `@example` and `@repl`.